src/components/views/articles/SingleArticle.js

```diff
@@ -7,9 +7,9 @@ import { connect } from "react-redux";
 import PropTypes from "prop-types";
 import { bindActionCreators } from "redux";
 import { Link } from "react-router-dom";
-
-import * as actionGenerators from "../../../actions/articlesActions";
 import HeaderComponent from "../../containers/headers/index";
+import * as actionGenerators from "../../../actions/articlesActions";
```

Comment:
Why not name the actionGenerators as articleActions? It would be easier to them as actions related to articles, assuming more than one set of articles are being imported.